app/partials/project-modal-editor.cjsx

```diff
@@ -78,11 +78,34 @@ ProjectModalEditor = createReactClass
       console.log 'ProjectModalEditor onStepOrderChange', arguments
 
   getInitialState: ->
+    error: ''
+    frequency: if @props.projectModal?.configuration?.minicourse_frequency then @props.projectModal?.configuration?.minicourse_frequency.join "," else ''
     stepToEdit: 0
 
   onClick: (stepIndex) ->
     @setState stepToEdit: stepIndex
 
+  changeFrequency: (e) ->
+    if @state.error then @setState { error: '' }
+    @setState { frequency: e.target.value }
+
+  saveFrequency: (e) ->
+    e.preventDefault()
+    if @state.error then @setState { error: '' }
+    allowedString = /^(\d(,\d)*)$/
```

Comment:
This was addressed in #5789 with an [adjustment](https://github.com/zooniverse/Panoptes-Front-End/pull/5789#issuecomment-682913091) to use the word "whole number" rather than "digit," but perhaps it still isn't clear? 

I think updating the instructions is a good idea, as I was initially confused about that same point.

Reply:
Maybe "single digit whole number", "whole number less than 10", or define that a frequency must be a "... single digit whole number..." in it's own sentence then can refer to the frequency number without noting "whole"? Not blocking at all, I'll defer to whatever think is best and approve.